google-cloud-clients/google-cloud-websecurityscanner/src/test/java/com/google/cloud/websecurityscanner/it/v1beta/VpcNegativeTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.websecurityscanner.it.v1beta;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+import com.google.api.gax.core.FixedCredentialsProvider;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.websecurityscanner.v1beta.CreateScanConfigRequest;
+import com.google.cloud.websecurityscanner.v1beta.ListScanConfigsRequest;
+import com.google.cloud.websecurityscanner.v1beta.ProjectName;
+import com.google.cloud.websecurityscanner.v1beta.ScanConfig;
+import com.google.cloud.websecurityscanner.v1beta.WebSecurityScannerClient;
+import com.google.cloud.websecurityscanner.v1beta.WebSecurityScannerClient.ListScanConfigsPagedResponse;
+import com.google.cloud.websecurityscanner.v1beta.WebSecurityScannerSettings;
+import com.google.common.collect.Lists;
+import java.io.FileInputStream;
+import java.io.IOException;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Negative Integration tests for VPC-SC. */
+@RunWith(JUnit4.class)
+public class VpcNegativeTest {
```

Comment:
Can you rename this to `VPCServiceControlNegativeTest` (and rename the file)? It will help us divide up our tests.

Reply:
Yes I renamed it to VPCServiceControlNegativeTest.